Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM alpine:latest
-RUN mkdir /app
-COPY . /app
+FROM alpine:3.19 as builder
```

Comment:
First off, thanks for putting in a pull request. I think overall you've made a good point that we should try to keep the container small.

As for pinning the version, there is always going to be a tradeoff between stability and security, and people are always going to argue about which is more important. I deliberately opted for latest because I want all the patches.